18F/identity-idp
# Modified CTA Update button for Spanish

## 🎫 Ticket
[LG-10085](https://cm-jira.usa.gov/browse/LG-10085) Implementation: Change the CTA "Update" for Spanish translations for the 6 views listed below

## 🛠 Summary of changes
Change the "Update" CTA button in Spanish from "Actualización" to "Actualizar" on these 6 pages:

Update the information on your ID, Update your current address, Update your SSN for In Person...
- [ ] es/verify/in_person/state_id
- [ ] es/verify/in_person/address
- [ ] es/verify/in_person/ssn

Update your current address, Update your SSN for Doc Upload...
- [ ] es/verify/address
- [ ] es/verify/ssn

Update your password...
- [ ] es/manage/password

## 📜 Testing Plan
Each check box corresponds with a view above, 6 checkboxes below/6 views above. 

1. Create an account. Confirm email. Set up a password. Set up Authentication method. (If you pick codes be sure to save them as you will need them to change your password.)
2. Change url to /verify to Get started verifying your identity.
3. Upload photos to fail proofing and proceed to Try in person.
4.  Fill out forms until you land on url `/verify/in_person/verify_info`
- [ ] Click update for State-issued ID.  When on `/es/verify/in_person/state_id` confirm the button has been updated to "Actualizar".
- [ ] Click update for Current residential address. When on `/es/verify/in_person/address` confirm the button has been updated to "Actualizar".
- [ ] Click update for Social Security Number. When on `/es/verify/in_person/ssn` confirm the button has been updated to "Actualizar".
5. Click Cancel and Start over.
6. Restart the flow to verify your identity. This time, upload photos to pass proofing.
7. Enter a ssn to eventually get to `/verify/verify_info`
- [ ] Click update for the address. When on `/es/verify/address` confirm the button has been updated to "Actualizar".
- [ ] Click update for Social Security Number. When on `/es/verify/ssn` confirm the button has been updated to "Actualizar".
8. Change the url to `/account` and click on Edit password under Your Account. You will be taken to `/manage/password`
9. Edit url to `es/manage/password`
- [ ] Confirm the button has been updated to "Actualizar".
10. Poke around for the Update button. There should not be more Update buttons in application and thus nothing else was updated as a result of this change that was not mentioned above.

##   ✅ Approvals
The original request was for 3 views. It was discovered that 3 more views would be affected.  Rather than only modifying the CAT Update in Team Joy's views only, Carmen asked Team Ada and Katherine for approval so that all of the buttons would be consistent. Both teams gave approval, see [LG-10085](https://cm-jira.usa.gov/browse/LG-10085) description for slack threads/approvals.


## 👀 Screenshots

<details>

<summary>es/verify/in_person/state_id</summary>

<img width="1406" alt="Screenshot 2023-07-07 at 1 55 04 PM" src="https://github.com/18F/identity-idp/assets/125507397/40c4b745-b41e-4f39-94f8-97760c91a771">

</details>

<details>

<summary>es/verify/in_person/address</summary>


<img width="687" alt="Screenshot 2023-07-07 at 1 55 45 PM" src="https://github.com/18F/identity-idp/assets/125507397/1e33d1a2-a99e-49c8-a51a-8cb969092eab">

</details>


<details>

<summary>es/verify/in_person/ssn</summary>

<img width="1352" alt="Screenshot 2023-07-07 at 2 02 34 PM" src="https://github.com/18F/identity-idp/assets/125507397/aea5d55f-1e99-4368-bf3e-fc4537cc5830">

</details>



<details>

<summary>es/verify/address</summary>

<img width="1391" alt="Screenshot 2023-07-07 at 2 04 03 PM" src="https://github.com/18F/identity-idp/assets/125507397/ee1d89d5-e1e1-4a15-9d30-875dffd04661">


</details>


<details>

<summary>es/verify/ssn</summary>

<img width="1297" alt="Screenshot 2023-07-07 at 2 05 45 PM" src="https://github.com/18F/identity-idp/assets/125507397/9ea94efc-e688-4503-994a-7c2bcf9cb07a">


</details>


<details>
<summary>/es/mange/password</summary>

<img width="1472" alt="Screenshot 2023-07-07 at 1 51 55 PM" src="https://github.com/18F/identity-idp/assets/125507397/5ac88417-4def-4be9-b49d-923d7d3661ad">

</details>
